src/_locales/fr/messages.json

```diff
@@ -196,7 +196,7 @@
     "hash": "4775fb2f88fd33d02b871e5021dec3e9"
   },
   "translatorUsername": {
-    "message": "PoorPocketsMcNewHold et le-jun",
+    "message": "PoorPocketsMcNewHold, le-jun et amcwb/stars",
```

Comment:
In any case, I guess you'd need to adjust the credits here then?

And if you are good, you can adjust the `CONTRIBUTORS.md` file in the root.